app/templates/components/ui-table/cell/events/view/tickets/attendees/cell-price.hbs

```diff
@@ -1,8 +1,26 @@
-<div class="weight-400">
-  {{currency-symbol paymentCurrency}}  {{format-number record.order.amount}}
-</div>
 {{#if record.order.discountCode.code}}
-  <div class="ui mini yellow label">
-    {{record.order.discountCode.code}}
+  {{#each record.ticket.discountCodes as |discountCode|}}
+    {{#if (eq discountCode.code record.order.discountCode.code)}}
```

Comment:
this is too much logic inside a template file, use a helper to get the discount code for an order, always keep minimal logic inside templates.

Reply:
@CosmicCoder96 I tried writing a helper but Idk why the ticket and order object are not returning value of attributes even. Not even basic attributes ( I checked in local by printing console logs).
Where am I doing it wrong?
@uds5501 @mrsaicharan1 @kushthedude maybe you people can help?

Reply:
I'll fix the code and write a helper test once I fix the logic

Reply:
@niranjan94 @CosmicCoder96 can you please tell me where am I doing wrong in the helper I added
> I tried writing a helper but Idk why the ticket and order object are not returning value of attributes even. Not even basic attributes ( I checked in local by printing console logs).

Reply:
@CosmicCoder96 I'm able to get discount value when printed the logs. But the FE shows `[object promise]`. What does this mean?
How can I return resolved value of promise?
![Screenshot_2019-06-15 All Attendees Tickets Order Events Open Event](https://user-images.githubusercontent.com/25428397/59549644-44892280-8f7e-11e9-9282-618539b1fb9f.png)